src/styles.css

```
@@ -154,9 +176,36 @@ body{background:var(--bg);color:var(--ink);font-family:'Plus Jakarta Sans',sans-
 .grid{display:grid;grid-template-columns:repeat(auto-fill,minmax(310px,1fr));gap:12px;margin-bottom:60px}
 
 /* ── CARD ── */
-.org-card{background:var(--card-bg);border:1.5px solid var(--border);border-radius:14px;padding:18px;cursor:pointer;position:relative;overflow:hidden;animation:fadeUp .25s ease forwards;transition:border-color .18s,box-shadow .18s,transform .15s,background .3s}
+.org-card{
+ background:
+  linear-gradient(
+    145deg,
+    rgba(255,255,255,.85),
```

Stylelint (17.10.0)

[error] Expected "rgba" to be "rgb" (color-function-alias-notation)

(color-function-alias-notation)